src/claimApplication/disability/information.js

```diff
@@ -10,8 +11,8 @@ export const id = "disabilityInformation";
 
 export const identifyingContent = {
   id,
-  elementId: 'ContentPlaceHolder1_ClaimantDisabilityTab_ClientState',
-  value: '"TabState":[true,false,false,false,false,false]',
+  elementId: 'ContentPlaceHolder1_ClaimantDisabilityTab_Dis1_dvShowFDD',
+  text: 'Select the date your disability',
```

Comment:
The reason I needed to edit this `identifyingContent` was a little weird. My Cypress test of the Back button was failing because this page's title was not updating after Back was pressed. Turns out actually *none* of the changes on this page were applied in that case, and haven't been for a little while! Glad I tossed in that test way back when. With this edit, the page is correctly identified and the changes applied.